### ISSUE TITLE
Placeholder: upcoming internal stats changes for 1.1

### ISSUE THREAD

**Issue by mark-rushakoff** [influxdata/docs.influxdata.com#780]
I'm pretty sure these are all the upcoming stats changes for 1.1 as of today.
# OSS
- [new `walPath` tag on shard stats](https://github.com/influxdata/influxdb/pull/7383)
- [active compaction stats](https://github.com/influxdata/influxdb/pull/7281) (not yet merged into 1.1)
- [max-values-per-tag config option](https://github.com/influxdata/influxdb/pull/7430): `writePointsDropped` stats on shard measurement as well as `pointsWrittenDropped` on http measurement are incremented when a point is dropped from a write.
# Enterprise
- `hh_processor.queueBytes` is an absolute value that indicates how many bytes remain in the queue. The number should remain correct across restarts, which is an improvement over the existing `bytesRead` and `bytesWritten` fields.
- `hh_processor.writeDropped` increments for every point that was read from the HH queue and appeared to be corrupted and was therefore dropped.

**Comment by mark-rushakoff** [comment]
tsm1_cache: writeOk, writeErr, writeDropped will be recorded as part of https://github.com/influxdata/influxdb/pull/7533.

**Comment by beckettsean** [comment]
waiting on https://github.com/influxdata/docs.influxdata.com/pull/777

**Comment by stevebang** [comment]
Closing as _internal metrics will not be documented -- intended for internal / support use.
